resources/assets/js/jsx/profile_page.jsx

```diff
@@ -53,16 +53,23 @@
 					dataType: 'json'
 			})
 			.done(function(userData) {
+				this.props.user.cover.id = userData.data.cover.id;
```

Comment:
this and the following line shouldn't be necessary. Additionally, props may never be modified.